hadoop-hdds/server-scm/src/test/java/org/apache/hadoop/hdds/scm/pipeline/TestPipelineManagerImpl.java

```diff
@@ -131,12 +133,12 @@ public void testCreatePipeline() throws Exception {
         createPipelineManager(true, buffer1);
     Assert.assertTrue(pipelineManager.getPipelines().isEmpty());
     Pipeline pipeline1 = pipelineManager.createPipeline(
-        HddsProtos.ReplicationType.RATIS, HddsProtos.ReplicationFactor.THREE);
+        new RatisReplicationConfig(ReplicationFactor.THREE));
     Assert.assertEquals(1, pipelineManager.getPipelines().size());
     Assert.assertTrue(pipelineManager.containsPipeline(pipeline1.getId()));
 
     Pipeline pipeline2 = pipelineManager.createPipeline(
-        HddsProtos.ReplicationType.RATIS, HddsProtos.ReplicationFactor.ONE);
+        new RatisReplicationConfig(ReplicationFactor.THREE));
```

Comment:
I think there is a mistake here - the old code had factor ONE. New code has THREE.

Reply:
Good point, thanks. Fixing it. (BTW, it seems to be the limitation of the unit test. Pipeline parameters are not checked, so it can be ONE/THREE, but let's keep it as it was)